src/pages/GovPages/Card.tsx

```diff
@@ -69,19 +70,28 @@ export const GovCard = ({ currency, currency1, apy, proposals }: CardProps) => {
   const theme = useContext(ThemeContext)
   const router = useRouter()
 
-  const onClick = () => {
+  const onCurrencyClick = () => {
```

Comment:
This and the handler below could probably be replaced by a `<Link />` component, cleaning up the code a bit in the process (the link needs to be undecorated, and for that I would create a dedicated component to be reused, since I actually found it useful to use in the farming stuff too).

Reply:
The card itself has got many styling including the ::before tag.
I think it's better to keep as now rather than styling <a> again.